news/_posts/2020-03-bug-bounty-program.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "Let the hunt begin"
+subtitle: "It's time to make PrestaShop better that it already is!"
+date:   2020-03-11 08:00:00
+authors: [ PierreRambaud, PrestaShop ]
+icon: icon-chat
+image: /assets/images/theme/meta-logo-build.png
+tags: [bug, bounty, program, maintenance]
+---
+
+Protecting the privacy of PrestaShop's users is one of our top priority. We build the software with this goal in mind.
+That's why we decided to welcome help from the outisde through a bounty program to put our security to the test!
+
+# Why this program
+
+Bug bounties are used by many leading companies to improve the security of their products.
```

Comment:
```suggestion
Bug bounties are used by many leading companies to improve the security of their softwares.
```

Reply:
It's not related to softwares only, the general purpose is about products.

Reply:
all our products are softwares.
but the bug bounty is also about the open source project, which is not really a product.

Reply:
We are not talking about our products in this line, it's about leading companies.